src/RawModelExportItem.tsx

```diff
@@ -0,0 +1,68 @@
+import Divider from '@mui/material/Divider'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/system/Box'
+import Button from '@mui/material/Button'
+
+import { useState } from 'react'
+import { useRouter } from 'next/router'
+import DirectoryTreeView from './common/DirectoryTreeView'
+import SplitButton from './common/SplitButton'
+
+function RawModelExportItem({ deploymentUuid, version }: { deploymentUuid: string; version: any }) {
```

Comment:
Is there a reason why version is typed as any? You're using it as a key below so I'm assuming it's a string?

Reply:
I believe this was a holdover from when it was using the parent variable (confusingly also named version), and I missed that when I decided to use the child variable rather than clutter the code with version.version everywhere